virtool/users/sessions.py

```diff
@@ -181,6 +181,14 @@ async def get_authenticated(self, session_id: str, session_token: str) -> Sessio
 
         return Session(**session)
 
+    async def validate_session(self, session_id: str):
+        """
+        Validates a session with a given ``session_id``.
+
+        :param session_id: the session id
+        """
+        await self._get(session_id)
+
```

Comment:
Good direction, couple of tweaks:

 1. rename along the lines of `check_authenticated`
 2.  return true if session is authenticated, return false if the session does not exist or is not authenticated 

I would avoid throwing errors in this flow as you can give a valid answer regardless of of the passed `session_id` exists or not